pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
Also, I should note that when I run the code, `--hook-stage` variable is passed to the `run` method.